containers/payments/subscription/BlackFridayModal.js

```diff
@@ -32,19 +40,19 @@ const BlackFridayModal = ({ bundles = [], onSelect, ...rest }) => {
 
     const BILLED_DESCRIPTION = ({ cycle, amount, notice }) =>
         ({
-            [MONTHLY]: c('Title').jt`Billed as ${amount} (${notice})`,
-            [YEARLY]: c('Title').jt`Billed as ${amount} (${notice})`,
-            [TWO_YEARS]: c('Title').jt`Billed as ${amount} (${notice})`
+            [MONTHLY]: c('Title').jt`Billed as ${amount} ${NOTICES[notice]} for 1 month`,
```

Comment:
I would have the asterisks by the end: `Billed as ${amount} for 1 month ${NOTICES[notice]}`. I think they are also at the end in the proposal